src/alertsmanagement/azext_alertsmanagement/_help.py

```diff
@@ -0,0 +1,67 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=too-many-lines
+# pylint: disable=line-too-long
+from knack.help_files import helps  # pylint: disable=unused-import
+
+
+helps['alertsmanagement action-rule'] = """
+    type: group
+    short-summary: Commands to manage alertsmanagement action rule.
+"""
+
+helps['alertsmanagement action-rule create'] = """
+    type: command
+    short-summary: Creates/Updates a specific action rule
+    examples:
+      - name: Create an action rule.
+        text: |-
+               az alertsmanagement action-rule create --resource-group "alertscorrelationrg" --name "rule" --location "Global" --status "Enabled" --rule-type Suppression --severity Equals Sev0 Sev2 --recurrence-type Daily --start-date 12/09/2018 --end-date 12/18/2018 --start-time 06:00:00 --end-time 14:00:00
+
+"""
+
+helps['alertsmanagement action-rule update'] = """
+    type: command
+    short-summary: Creates/Updates a specific action rule
+    examples:
+      - name: Update an action rule.
+        text: |-
+               az alertsmanagement action-rule update --resource-group "alertscorrelationrg" --name \\
+               "WeeklySuppression" --status "Disabled"
+"""
+
+helps['alertsmanagement action-rule delete'] = """
+    type: command
+    short-summary: Delete a given action rule
+    examples:
+      - name: DeleteActionRule
```

Comment:
This name is not consistent with other example names.

Reply:
Done